components/layout/block.rs

```diff
@@ -1491,7 +1491,7 @@ impl BlockFlow {
         debug_assert!(!self.is_inline_flex_item());
 
         // Compute the available space for us, based on the actual floats.
-        let rect = self.base.floats.available_rect(Au(0),
```

Comment:
I'm pretty sure they are fixed by force reflow because all the blocks in these two tests have zero margins.